### PR TITLE
Master account usability jao

### PR DESCRIPTION
**Pull request by jao-odoo**
Task : https://www.odoo.com/web#id=40157&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

Pad : https://pad.odoo.com/p/r.ce7e9384c5fffa227d7ed5cb9cf5c22c
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/payment/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,5 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 payment_acquirer_all,payment.acquirer.all,model_payment_acquirer,,1,0,0,0
+payment_acquirer_account_user,payment.acquirer.non_adviser,model_payment_acquirer,account.group_account_user,1,0,0,0
 payment_acquirer_user,payment.acquirer.user,model_payment_acquirer,base.group_user,1,1,1,0
```
Comment (fmdl) [inline addons/payment/security/ir.model.access.csv]: Strange : `account.group_account`_user include `base.group_user`

**Comment by rgo-odoo** [comment]
LGTM

File: addons/account_voucher/__manifest__.py [hunk 2]
```diff
@@ -2,26 +2,11 @@
...
-You could use this simplified accounting in case you work with an (external) account to keep your books, and you still want to keep track of payments. 
-
-The Invoicing system includes receipts and vouchers (an easy way to keep track of sales and purchases). It also offers you an easy method of registering payments, without having to encode complete abstracts of account.
-
-This module manages:
-
-* Voucher Entry
-* Voucher Receipt [Sales & Purchase]
-* Voucher Payment [Customer & Vendors]
+This module allows you to record sales and purchases receipts. Receipts are useful when the payment is done directly. Thanks to the receipts, no need to encode an invoice and a payment, the receipt is enough.
```
[the remark this answers is not in the document]
Comment (jao-odoo) [reply addons/account_voucher/__manifest__.py]: will improve the string

File: addons/payment/security/ir.model.access.csv [hunk 3]
```diff
@@ -1,6 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 payment_acquirer_all,payment.acquirer.all,model_payment_acquirer,,1,0,0,0
-payment_acquirer_user,payment.acquirer.user,model_payment_acquirer,base.group_user,1,1,1,0
+payment_acquirer_account_user,payment.acquirer.non_adviser,model_payment_acquirer,account.group_account_user,1,0,0,0
+payment_acquirer_user,payment.acquirer.user,model_payment_acquirer,base.group_user,1,1,1,1
```
[the remark this answers is not in the document]
Comment (jao-odoo) [reply addons/payment/security/ir.model.access.csv]: @oco-odoo I can't recall why I gave deletion rights, so I will revert the change. And do you want me to restrict regular users to read and/or write rights only ?

File: addons/payment/security/ir.model.access.csv [hunk 4]
```diff
@@ -1,6 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 payment_acquirer_all,payment.acquirer.all,model_payment_acquirer,,1,0,0,0
-payment_acquirer_user,payment.acquirer.user,model_payment_acquirer,base.group_user,1,1,1,0
+payment_acquirer_account_user,payment.acquirer.non_adviser,model_payment_acquirer,account.group_account_user,1,0,0,0
```
[the remark this answers is not in the document]
Comment (jao-odoo) [reply addons/payment/security/ir.model.access.csv]: alright

File: addons/account/views/res_config_settings_views.xml [hunk 5]
```diff
@@ -381,16 +376,11 @@
...
+                                <div class="o_setting_right_pane" id="budget_management">
                                     <label for="module_account_budget"/>
                                     <div class="text-muted">
                                         Compare actual revenues &amp; costs with budgets
                                     </div>
-                                    <div id="msg_account_budget" class="content-group" attrs="{'invisible': [('module_account_budget', '=', False)]}">
-                                        <div class="text-warning mt16 mb4">
-                                            Save this page and come back here to set up the feature.
-                                        </div>
-                                    </div>
```
[the remark this answers is not in the document]
Comment (jao-odoo) [reply addons/account/views/res_config_settings_views.xml]: it's there in spec in <a href="https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrckhuNHlydE5TdnM/view?usp=drivesdk">point 4</a> :)

File: addons/payment/security/ir.model.access.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (oco-odoo) [reply addons/payment/security/ir.model.access.csv]: To me, they should only have read rights, but perhaps there is a good reason to the fact they have more than that... @qdp-odoo what do you think?

File: addons/account/views/res_config_settings_views.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (oco-odoo) [reply addons/account/views/res_config_settings_views.xml]: It is for followup, but not for budget management. Or is there a reference somewhere else adding something to it?

File: addons/payment/security/ir.model.access.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (qdp-odoo) [reply addons/payment/security/ir.model.access.csv]: same here

File: addons/account/views/res_config_settings_views.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (jao-odoo) [reply addons/account/views/res_config_settings_views.xml]: it was asked on `3rd January 2018` review on the task pad saying to remove it from `revenue config` as well, https://drive.google.com/file/d/1lXhpfCrDssgaqzm-ACIC6EBxXVwQBraB/view?usp=drivesdk

[the remark this answers is not in the document]
Comment (oco-odoo) [reply addons/account/views/res_config_settings_views.xml]: Okay, I hadn't realized this setting was linked to that ^^ Thanks!

File: addons/account_check_printing/models/account_payment.py [hunk 6]
```diff
@@ -86,10 +86,14 @@ def print_checks(self):
         if not self[0].journal_id.check_manual_sequencing:
             # The wizard asks for the number printed on the first pre-printed check
             # so payments are attributed the number of the check the'll be printed on.
-            last_printed_check = self.search([
-                ('journal_id', '=', self[0].journal_id.id),
-                ('check_number', '!=', 0)], order="check_number desc", limit=1)
```
Comment (tivisse) [inline addons/account_check_printing/models/account_payment.py]: And why not ``('check_number', '!=', "0")``

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**